src/lang/base/Syntax.ml

```diff
@@ -203,6 +203,110 @@ type literal =
 [@@deriving sexp]
 
 
+(* Builtins *)
+type builtin =
+  | Builtin_eq
+  | Builtin_concat
+  | Builtin_substr
+  | Builtin_strlen
+  | Builtin_to_string
+  | Builtin_blt
+  | Builtin_badd
+  | Builtin_bsub
+  | Builtin_dist (* in cashflow checker only *)
```

Comment:
This should be removed. We removed the `dist` built-in about a month ago, but apparently we forgot to remove it from the cashflow checker.